frontend/stories/buttons/Button.stories.tsx

```diff
@@ -57,21 +55,9 @@ export default {
 } as ComponentMeta<typeof Button>;
 
 // More on component templates: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
-const Template: ComponentStory<typeof Button> = (args) => <Button {...args}>Material ui button</Button>;
+const Template: ComponentStory<typeof Button> = (args) => <Button {...args}>Controls Button</Button>;
 
-export const Primary = Template.bind({});
 // More on args: https://storybook.js.org/docs/react/writing-stories/args
-Primary.args = {
```

Comment:
Any reason the 'outlined' examples have been removed?
Also you've removed the 'label' and 'label-outlined' variants from the options, so once you change the variant of one of these you can't change back